.travis.yml

```diff
@@ -0,0 +1,18 @@
+sudo: false
+language: node_js
+node_js:
+- '6'
+env:
+  global:
+  - SAUCE_USERNAME: dojo2-ts-ci
+  - SAUCE_ACCESS_KEY: e92610e3-834e-4bec-a3b5-6f7b9d874601
+  - FORCE_COLOR: 1
+install:
+- travis_retry npm install
+script:
+- npm run test
```

Comment:
Make this `npm run test && npm run uploadCoverage` so it won't run `uploadCoverage` if `test` fails

Reply:
Also, run `npm run lint` before `npm run test`